Dockerfile

```diff
@@ -1,14 +1,16 @@
 FROM node:16-alpine@sha256:72a490e7ed8aed68e16b8dc8f37b5bcc35c5b5c56ee3256effcdee63e2546f93 AS builder
 MAINTAINER Tyler Levine <tyler@bitgo.com>
 RUN apk add --no-cache git python3 make g++ libtool autoconf automake
-COPY --chown=node:node . /tmp/bitgo/
-WORKDIR /tmp/bitgo/modules/express
-USER node
+WORKDIR /tmp/bitgo
+COPY . .
 RUN \
+    # clean up unnecessary local node_modules and dist
+    rm -rf **/node_modules **/dist && \ 
     # install with dev deps so we can run the prepare script
     yarn install --frozen-lockfile && \
```

Comment:
With it not being dependent on other packages internally, but from npm, do we need two `yarn install`s still?

Reply:
@mmcshinsky-bitgo the purpose of this ticket is to make it built with other local packages so we need to use yarn. Also, the second yarn install is to reduce the size of the root `node_modules` by getting rid of devDependencies packages after the required packages i.e bitgo, sdk-core, and etc were built.